Source/Common/ContextJsonConverter.cs

```diff
@@ -11,96 +11,81 @@ namespace Schema.NET
+                throw new ArgumentNullException(nameof(options));
             }
 #endif
+            var context = new JsonLdContext();
 
-            var context = hasExistingValue ? existingValue! : new JsonLdContext();
-
-            string? name;
-            string? language;
-            if (reader.TokenType == JsonToken.String)
+            string? name = null;
+            string? language = null;
+            if (reader.TokenType == JsonTokenType.String)
             {
-                name = (string?)reader.Value;
-                language = null;
+                name = reader.GetString();
             }
-            else if (reader.TokenType == JsonToken.StartObject)
+            else if (reader.TokenType == JsonTokenType.StartObject)
             {
-                var o = JObject.Load(reader);
+                var document = JsonDocument.ParseValue(ref reader);
 
-                var nameProperty = o.Property("name", StringComparison.OrdinalIgnoreCase);
-                name = nameProperty?.Value?.ToString() ?? "https://schema.org";
+                if (document.RootElement.TryGetProperty("name", out var nameElement))
+                {
+                    name = nameElement.GetString() ?? Constants.HttpsSchemaOrgUrl;
+                }
 
-                var languageProperty = o.Property("@language", StringComparison.OrdinalIgnoreCase);
-                language = languageProperty?.Value?.ToString();
+                if (document.RootElement.TryGetProperty("@language", out var languageElement))
+                {
+                    language = languageElement.GetString();
+                }
             }
             else
             {
-                var a = JArray.Load(reader);
+                var array = JsonDocument.ParseValue(ref reader).RootElement.EnumerateArray();
 
-                name = language = null;
-                foreach (var entry in a)
+                foreach (var entry in array)
                 {
-                    if (entry.Type == JTokenType.String)
+                    if (entry.ValueKind == JsonValueKind.String)
                     {
-                        name ??= (string?)entry;
+                        name ??= entry.GetString();
                     }
-                    else
+                    else if (entry.ValueKind == JsonValueKind.Object)
                     {
-                        var o = (JObject)entry;
-
-                        var nameProperty = o.Property("name", StringComparison.OrdinalIgnoreCase);
-                        name ??= nameProperty?.Value?.ToString() ?? "https://schema.org";
-
-                        var languageProperty = o.Property("@language", StringComparison.OrdinalIgnoreCase);
-                        language ??= languageProperty?.Value?.ToString();
+                        if (entry.TryGetProperty("name", out var nameElement))
+                        {
+                            name ??= nameElement.GetString() ?? "http://schema.org";
```

Comment:
Should we use the constant here?

Reply:
Fixed! Missed this when merging the code.